src/test/java/org/synyx/urlaubsverwaltung/mail/MailServiceImplTest.java

```diff
@@ -207,6 +208,38 @@ void sendTechnicalMail() {
         verify(mailSenderService).sendEmail(eq("no-reply@firma.test"), eq(singletonList(to)), eq("subject"), eq("emailBody"));
     }
 
+    @Test
+    void sendMailToWithNotificationAndPersonsAndAdministrator() {
+
+        when(mailProperties.getAdministrator()).thenReturn("admin@firma.test");
+
+        setupMockServletRequest();
+
+        final Person hans = new Person();
+        hans.setEmail("hans@firma.test");
+        when(personService.getPersonsWithNotificationType(NOTIFICATION_USER)).thenReturn(List.of(hans));
+
+        final Person franz = new Person();
+        franz.setEmail("franz@firma.test");
+
+        final String subjectMessageKey = "subject.overtime.created";
+        final String templateName = "overtime_office";
+
+        final Mail mail = Mail.builder()
+            .withRecipient(true)
```

Comment:
Without IntelliJ's tooltips it's hard to recognize what the value `true` means here. Why not rename the method to `withTechnicalRecipient()` (without Parameter). But maybe I miss some advantage of overloading here.